src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.ts

```diff
@@ -8,61 +13,99 @@ import { FormControl, FormGroup, Validators } from '@angular/forms';
 })
 export class UbsClientProfilePageComponent implements OnInit {
   userForm: FormGroup;
+  public googleIcon = SignInIcons.picGoogle;
+  public editing = false;
```

Comment:
add 'is' prefix for booling vars
so editing will be isEditing

Reply:
done